ted/src/main/java/frc/robot/subsystems/ShooterOutfeedSubsystem.java

```diff
@@ -164,6 +164,11 @@ private void configureOutfeedMotors() {
     // do not config feedbacksource, since the default is the internal one.
     talonConfigs.Voltage.PeakForwardVoltage = 12;
     talonConfigs.Voltage.PeakReverseVoltage = -12;
+    // maximum current settign
+    talonConfigs.CurrentLimits.StatorCurrentLimit = Constants.shooterStatorCurrentMaximumAmps;
+    talonConfigs.CurrentLimits.StatorCurrentLimitEnable = true;
+    talonConfigs.CurrentLimits.SupplyCurrentLimit = Constants.shooterSupplyCurrentMaximumAmps;
+    talonConfigs.CurrentLimits.SupplyCurrentLimitEnable = true;
```

Comment:
We also need voltageConfigs.withSupplyVoltageTimeConstant(0.02);

Reply:
updated